### ISSUE TITLE
RC not recognized anymore with 3.0.1

### ISSUE THREAD

**Issue by Caipi77** [MegaPirateNG/ardupilot-mpng#1]
Hi,
I just uploaded MPNG3.0.1R2 to my HK board and my RC is not recognized anymore. I flew successfully with MPNG2.9 and now this issue occurred when I tried to switch to 3.0.1. I used Arduino 1.0.3 and Mission planner 1.2.72mav1.0.

Is there anything I have to change on the hardware if I switch from AC2.9 to AC3.0.1?

Thanks a lot for your help!

Sven

**Comment by SirAlex** [comment]
Are you properly configured 3.0.1 regarding RC? Check this posts http://www.megapirateng.com/how-to/
If nothing helped, post full information about your hardware configuration.

**Comment by Caipi77** [comment]
Thanks Alex, I got it working with your post. 
I did a motor test on the ground before actually flying and unfortunately I encountered the bug that you posted on 30/08/2013. The board hangs up at some Throttle level around 40-50%. 
This is my configuration:
- Quadrocopter
- HK board (analogue Crius V1), running on regular PPM (no sum). 
  The board doesn't permit data logging, is there any more information I can provide to you!
  Thanks a lot for your support!
  Sven
